10x_Experimental_Scenarios.md

```diff
@@ -7,87 +7,124 @@ The relationship between **biological samples** (e.g. a cell suspension extracte
+This document is only necessary for feature barcode (FBM) runs:
+- `id`: unique id for the feature (can't collide with gene name)
+- `name`: human readable feature name (e.g. gene name)
+- `read`: specifies which sequencing read contains the sequence (e.g. R2)
+- `pattern`: specifies how to extract seq from read
+- `feature_type`: (e.g. custom <!-- more info? -->)
+- `target_gene_id`, `target_gene_name`: optional, CRISPR-specific.
 
-# Proposed Processing-Run Spreasheets
-Here we are defining a `Processing-Run` to consist of a set of Cell Ranger jobs performed on one or more BCL files to obtain one or more FBMs and/or TCR/BCR sequence analysis. These are the spreadsheets that are planned to be used as inputs for a specific `Processing-Run` to the rebuilt pipeline. They will be similar to the input spreadsheets that are directly given as arguments to Cell Ranger Mkfastq and Count, but will contain the additional information required all arguments (e.g. expected cell count) as well as a layout of how to run the full `Processing-Run` set of jobs.
+# `Processing-Run` Spreadsheets
+Here we define a `Processing-Run` as a set of Cell Ranger jobs performed on one or more BCL files to obtain one or more FBMs and/or TCR/BCR sequence analysis.
 
-We will require at most two CSVs from the 10x technicians: 
+A `Processing-Run` takes the spreadsheets below as inputs. They are similar to the spreadsheets that `cellranger mkfastq` and `count` take as inputs, but contain additional required arguments (e.g. expected cell count) as well as a layout of how to run the full `Processing-Run` set of jobs.
 
-### 1. Custom Sheet CSV
+## 1. Custom Sheet CSV
 
 | Lane| Sample | Index | Library Type | Reference Transcriptome | Number of Cells | Chemistry |
 |---|---|---|---|---|---|---|
 | 1  | S1_GEX | SI-GA-A3 | Gene Expression | GRCh38 | 3000 | 5-prime_V2 |
 
-This CSV will be used to construct the simple csv input for mkfastq as well as the libraries csv for count. In addition the last two columns will be used to construct the additional arguments for cellranger count.
+This CSV will be used to construct [the sample sheet CSV input for `mkfastq`][10X Sample Sheet CSV] as well as [the libraries CSV for `count`][10X Libraries CSV]. The last two columns will be used to construct additional arguments for `cellranger count`.
 
-### 2. Feature Reference CSV (exactly as required by count)
-We will have the 10x techs produce the feature reference CSV (presumably via lookup tables). 
+## 2. Feature Reference CSV (exactly as required by count)
+The 10x techs will produce the feature reference CSV (presumably via lookup tables). <!-- more info? -->
```

Comment:
more info?

Reply:
This is where we need to work with Laura to get a convention set up for feature names that is more easily parsable and contains at least: official gene symbol, chemistry type, chemistry version (possibly lot number). I propose an underscore delimited name. This name will link to a specific custom oligo name and sequence (e.g. ACTG).